larq/metrics.py

```diff
@@ -62,7 +62,49 @@ def get_training_metrics():
     return _GLOBAL_TRAINING_METRICS
 
 
-class FlipRatio(Metric):
+class LarqMetric(Metric):
+    """Metric with support for both 1.13 and 1.14+"""
+
+    def reset_states(self):
```

Comment:
We can also remove this method since it is implemented in the parent class already:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/metrics.py#L203-L209

Reply:
Agreed, had not noticed this